pkg/cloud-provider/cloudapi/common/cloud-mock_test.go

```diff
@@ -23,6 +23,7 @@ import (
 	reflect "reflect"
 
 	v1alpha1 "antrea.io/nephe/apis/crd/v1alpha1"
+	v1alpha10 "antrea.io/nephe/apis/runtime/v1alpha1"
```

Comment:
```suggestion
	runtimev1alpha1 "antrea.io/nephe/apis/runtime/v1alpha1"
```

Reply:
This is autogenerated code, i don't think editing is a good idea, let me know your thoughts.